bridge/slack/handlers.go

```diff
@@ -290,6 +298,27 @@ func (b *Bslack) handleMessageEvent(ev *slack.MessageEvent) (*config.Message, er
 	return &rmsg, nil
 }
 
+func (b *Bslack) handleTypingEvent(ev *slack.UserTypingEvent) (*config.Message, error) {
+	var err error
+	// use our own func because rtm.GetChannelInfo doesn't work for private channels
+	channelInfo, err := b.getChannelByID(ev.Channel)
+	if err != nil {
+		return nil, err
+	}
+
+	rmsg := config.Message{
+		Username: b.getUsername(ev.User),
+		Channel:  channelInfo.Name,
+		Account:  b.Account,
+		Event:    config.EVENT_USER_TYPING,
+		ID:       "slack " + time.Now().String(),
```

Comment:
Why setting an ID ?

Reply:
heh happy to leave out, just thought it was perhaps convention. I think I saw it in some debug output, but I'll double-check that.

We probably don't want to store these msgs in the cache, right?  (If I understand correctly, it could theoretically really inflate the cache size, and mean less history for edits/threading)

Related to that, would having a static ID or empty string help here, or should we have logic elsewhere to keep it out?